packages/website/package.json

```diff
@@ -13,13 +13,23 @@
 		"@remix-run/node": "^1.7.0",
 		"@remix-run/react": "^1.7.0",
 		"@remix-run/serve": "*",
+		"@remix-run/server-runtime": "^1.7.2",
 		"faunadb": "^4.6.0",
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
-		"remix-utils": "^3.3.0"
+		"react-icons": "^4.4.0",
+		"react-is": "^18.2.0",
+		"react-uuid": "^1.0.3",
+		"remix-utils": "^3.3.0",
+		"styled-components": "^5.3.6",
+		"uuid": "^9.0.0"
 	},
 	"devDependencies": {
-		"@remix-run/dev": "*"
+		"@remix-run/dev": "*",
+		"@types/react": "^18.0.21",
+		"@types/styled-components": "^5.1.26",
+		"eslint": "^8.24.0",
+		"typescript": "^4.8.4"
```

Comment:
Packages existed in the workspace is unnecessary here.

https://github.com/io-club/website/blob/2022/package.json